<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R7ALOTS" name="Default test" tests="5" failures="1" timestamp="27 Aug 2020 23:58:57 GMT" time="128.261" errors="0">
  <testcase name="UserLogin" time="10.383" classname="com.Guru99.BaseTest.Base$ValidateRegisterAccount"/>
  <testcase name="TC02" time="4.942" classname="com.Guru99.BaseTest.Base$ValidateRegisterAccount"/>
  <testcase name="TC03" time="4.073" classname="com.Guru99.BaseTest.Base$ValidateRegisterAccount"/>
  <testcase name="TC04" time="4.177" classname="com.Guru99.BaseTest.Base$ValidateRegisterAccount"/>
  <testcase name="TC05" time="12.98" classname="com.Guru99.BaseTest.Base$ValidateRegisterAccount">
    <failure type="java.lang.IllegalMonitorStateException">
      <![CDATA[java.lang.IllegalMonitorStateException
	at java.lang.Object.wait(Native Method)
	at com.Guru99.BaseTest.Base$ValidateRegisterAccount.TC05(Base.java:245)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC05 -->
</testsuite> <!-- Default test -->
